sites/partners/src/lib/listings/AdditionalMetadataFormatter.ts

```diff
@@ -71,19 +71,21 @@ export default class AdditionalMetadataFormatter extends Formatter {
       this.data.reviewOrderType = ReviewOrderTypeEnum.waitlist
     }
 
-    if (this.data.listingFeatures) {
+    if (this.data.accessibilityFeatures) {
```

Comment:
This was the main reason the feature was not working. We need to send an object to the backend, but the frontend form is looking for an array. By using a different field to store the values we can convert it to the correct format on submission